src/MultipleComplaintImageFactory.js

```diff
@@ -75,7 +75,11 @@ export default function(Complaints, SelectImages) {
           let { active } = state;
           let activeObject = state[active];
           let complaints = activeObject.complaints.concat({ pos: pointerPos });
-          let updatedItem = { [active]: { ...activeObject, complaints } };
+          let currentWidth = this.currentWidth;
+          let updatedItem = {
+            [active]: { ...activeObject, complaints },
+            srcWidth: currentWidth
```

Comment:
Do we only need to store the width?

Reply:
Yes. we are using width and height both as width only.

```
stage.width(width);
stage.height(width);
```

```
image.width(width - imagePadding);
image.height(width - imagePadding);

```